packages/core/addon/models/delivery-rule.js

```diff
@@ -34,7 +35,11 @@ export default DS.Model.extend(Validations, {
   updatedOn: DS.attr('moment'),
   deliveryType: DS.attr('string', { defaultValue: 'report' }),
   frequency: DS.attr('string', { defaultValue: 'week' }),
-  schedulingRules: DS.attr({ defaultValue: () => {return featureFlag('enabledNotifyIfData') ? {mustHaveData: false} : {} }}),
+  schedulingRules: fragment('fragments/scheduling-rules', {
+    defaultValue: () => {
```

Comment:
Do we need defaultValue here, if we define defaults in the fragment?

Reply:
we discussed this. Removed it from the fragment definition